recipes/rbdl-casadi/bld.bat

```diff
@@ -0,0 +1,13 @@
+mkdir build
+cd build
+
+cmake ../ ^
+    -GNinja ^
+    -DCMAKE_INSTALL_PREFIX="%PREFIX%" ^
+    -DRBDL_BUILD_STATIC=ON ^
```

Comment:
Try to build dynamic on Windows. Generally we prefer dynamic libraries where possible.

Reply:
Yep definitely, rbdl-casadi is actually a copy of rbdl in which I changed the backend for linear algebra (from Eigen to CasADi). Eigen did not allow for dynamic build on Windows, but I have no clue if it is possible with CasADi (I assume it is). My goal is indeed to dynamically build it. 

So far, I can't test this and having this file is just in preparation of future work. I can remove it if it makes things confusing

Thanks for the review